### PR TITLE
Mapping progress monitor: 'Tot deprecated unmapped' column

### PR DESCRIPTION
**Pull request by joeflack4**
Updates
```
Mapping progress monitor
- Update: docs/reports/unmapped.md: Added column 'Tot deprecated unmapped'
```

### CODE REVIEW

File: docs/reports/unmapped.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-| [NCIT](./unmapped_ncit.md)         | 174,300     | 150,309        | 5,055            | 18,936                                    | 6,728                     | 12,208                      | 64.5%                     |
-| [OMIM](./unmapped_omim.md)         | 28,659      | 17,286         | 1,345            | 10,028                                    | 9,779                     | 249                         | 2.5%                      |
-| [ORDO](./unmapped_ordo.md)         | 9,448       | 292            | 0                | 9,156                                     | 8,934                     | 222                         | 2.4%                      |
-| [DOID](./unmapped_doid.md)         | 13,648      | 2,599          | 2,467            | 11,048                                    | 9,748                     | 1,300                       | 11.8%                     |
+| Ontology                           | Tot terms   | Tot excluded   | Tot deprecated   | Tot deprecated unmapped   | Tot mappable _(!excluded, !deprecated)_   | Tot mapped _(mappable)_   | Tot unmapped _(mappable)_   | % unmapped _(mappable)_   |
```
Comment (joeflack4) [inline docs/reports/unmapped.md:2]: I re-ran mapping progress docs and got these results:
| Ontology                           | Tot deprecated   | Tot deprecated unmapped |
|:-----------------------------------|:-----------------|:--------------------------|
| [ICD10WHO](./unmapped_icd10who.md) | 0                | 0                         |
| [ICD10CM](./unmapped_icd10cm.md)   | 0                | 0                         |
| [NCIT](./unmapped_ncit.md)         | 5,055            | 5,022                     |
| [OMIM](./unmapped_omim.md)         | 1,345            | 1,255                     |
| [ORDO](./unmapped_ordo.md)         | 0                | 0                         |
| [DOID](./unmapped_doid.md)         | 2,467            | 2,448                     |

File: src/scripts/unmapped_docs.py [hunk 2]
```diff
@@ -58,6 +58,7 @@ def update_mapping_progress_in_docs():
             'Tot terms': f"{len(df):,}",
             'Tot excluded': f"{len(df[df['is_excluded'] == True]):,}",
             'Tot deprecated': n_deprecated,
+            'Tot deprecated unmapped': f"{len(df[(df['is_deprecated'] == True) & (df['is_mapped'] == False)]):,}",
```
Comment (joeflack4) [inline src/scripts/unmapped_docs.py:61]: Per [#226](https://github.com/monarch-initiative/mondo-ingest/issues/226#issuecomment-1468044153), I added a new column `Tot deprecated unmapped`.

I also double checked the logic to make sure that obsoletes/deprecated terms are not included in the "unmapped mappables". Indeed, it appears they are not:
```python
        mappable_df = df[(df['is_excluded'] == False) & (df['is_deprecated'] == False)]
        unmapped_mappable_df = mappable_df[mappable_df['is_mapped'] == False]
```

**Review by joeflack4 — COMMENTED** [review]
@matentzn @sabrinatoro This should, I believe, bring conclusion to #226. Take a look at the markdown table I included here as well.

File: docs/reports/unmapped.md (hunk 1 above)
Comment (joeflack4) [reply docs/reports/unmapped.md:2]: We have a high amount of terms that are deprecated/obsolete and mapped. I'm actually not sure though, is this number supposed to be low, high, or doesn't really matter? I'm guessing doesn't really matter.

**Review by matentzn — COMMENTED** [review]
I expected this column to drastically reduce the number of terms in the unmapped mappable column? It didn't?

File: src/scripts/unmapped_docs.py (hunk 2 above)
Comment (matentzn) [reply src/scripts/unmapped_docs.py:61]: Very unfortunate. I hoped these large numbers in the unmapped mappable column would somehow decrease..

Comment (joeflack4) [reply src/scripts/unmapped_docs.py:61]: Yeah, as I probably mentioned on our last call, I was not under the impression that this was bugged. All I did here was add a new column. It doesn't seem that there is a need to edit the calculations for 'mappable unmapped', so the number didn't change.

If you want, we can do additional checking. Like we could look through these TSVs / dataframes and just double check a bunch of terms and make sure they are being correctly marked as mapped, etc.

**Review by matentzn — APPROVED** [review]

**Pull request merged**